doc/man3/RAND_DRBG_generate.pod

```diff
@@ -0,0 +1,78 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_generate,
+RAND_DRBG_bytes
+- generate random bytes using the given drbg instance
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ int RAND_DRBG_generate(RAND_DRBG *drbg, unsigned char *out, size_t outlen,
+                        int prediction_resistance,
+                        const unsigned char *adin, size_t adinlen);
+
+ int RAND_DRBG_bytes(RAND_DRBG *drbg, unsigned char *out, size_t outlen);
```

Comment:
This functions (and probably others) are documented in the source. That documentation should probably get removed in favor of this one.

Reply:
I had already started removing the comment docs, but then I noticed that some of the comments contain internal documentation, wich can't be moved to the manual pages, like the fact that the `drbg->lock` needs to be taken if non-null. Also, there are a lot of internal functions which have comment docs. Tearing out the comment docs for the public functions but leaving the ones for the private functions would have destroyed the consistency of the source code documentation. So I hesitated to remove it. Just removed the big explanation parts which are now essentially in man7.